app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/processor/Fido2ProviderProcessorImpl.kt

```diff
@@ -124,10 +139,108 @@ class Fido2ProviderProcessorImpl(
         cancellationSignal: CancellationSignal,
         callback: OutcomeReceiver<BeginGetCredentialResponse, GetCredentialException>,
     ) {
-        // no-op: RFU
-        callback.onError(GetCredentialUnsupportedException())
+        cancellationSignal.setOnCancelListener {
+            callback.onError(GetCredentialCancellationException())
+            scope.cancel()
```

Comment:
You probably don't want to cancel the scope since, that will be unrecoverable. I am betting you want to cancel the job?

Reply:
Ah interesting, good question. Is that not what is supposed to happen? Shouldn't cancelling break you out of the flow and return you to the website? The `processCreateCredentialRequest` method in this class does the same thing - should I change it too?